Minor Site Updates

FoggyMtnDrifter opened the pull request:
This PR adds RISC-V International back to the partners page after it was noticed they were missing.

ressonix commented:
You need to add your language to `config.toml` and `vercel.json` before we can proceed. There are already examples in there you can reference.

rbouikila commented:
@ressonix i update config.toml and vercel.json file

FoggyMtnDrifter committed:
update export compliance matrix

FoggyMtnDrifter committed:
remove unused imports

FoggyMtnDrifter committed:
add crowdin configuration

FoggyMtnDrifter committed:
New translations en.json (Romanian)

FoggyMtnDrifter committed:
New translations en.json (French)

FoggyMtnDrifter committed:
New translations en.json (Spanish)

FoggyMtnDrifter committed:
New translations en.json (Afrikaans)

FoggyMtnDrifter committed:
New translations en.json (Arabic)

FoggyMtnDrifter committed:
New translations en.json (Catalan)

FoggyMtnDrifter committed:
New translations en.json (Czech)

FoggyMtnDrifter committed:
New translations en.json (Danish)

FoggyMtnDrifter committed:
New translations en.json (German)

FoggyMtnDrifter committed:
New translations en.json (Greek)

FoggyMtnDrifter committed:
New translations en.json (Finnish)

FoggyMtnDrifter committed:
New translations en.json (Hebrew)

FoggyMtnDrifter committed:
New translations en.json (Hungarian)

FoggyMtnDrifter committed:
New translations en.json (Italian)

FoggyMtnDrifter committed:
New translations en.json (Japanese)

FoggyMtnDrifter committed:
New translations en.json (Korean)

FoggyMtnDrifter committed:
New translations en.json (Dutch)

FoggyMtnDrifter committed:
New translations en.json (Norwegian)

FoggyMtnDrifter committed:
New translations en.json (Polish)

FoggyMtnDrifter committed:
New translations en.json (Portuguese)

FoggyMtnDrifter committed:
New translations en.json (Russian)

FoggyMtnDrifter committed:
New translations en.json (Serbian (Cyrillic))

FoggyMtnDrifter committed:
New translations en.json (Swedish)

FoggyMtnDrifter committed:
New translations en.json (Turkish)

FoggyMtnDrifter committed:
New translations en.json (Ukrainian)

FoggyMtnDrifter committed:
New translations en.json (Chinese Simplified)

FoggyMtnDrifter committed:
New translations en.json (Chinese Traditional)

FoggyMtnDrifter committed:
New translations en.json (English)

FoggyMtnDrifter committed:
New translations en.json (Vietnamese)

FoggyMtnDrifter committed:
New translations en.json (Portuguese, Brazilian)

FoggyMtnDrifter committed:
New translations en.json (Indonesian)

FoggyMtnDrifter committed:
New translations en.json (Persian)

FoggyMtnDrifter committed:
New translations en.json (Tamil)

FoggyMtnDrifter committed:
New translations en.json (Hindi)

FoggyMtnDrifter committed:
remove unneeded message files

FoggyMtnDrifter committed:
add riscv to partners

FoggyMtnDrifter committed:
Merge branch 'main' into develop

Grammaresque committed:
Create community-update-november-2024.md

press release about 9.5 release in progress, our RISC-V news, plus docs and community news.

Grammaresque committed:
Update community-update-november-2024.md

add words "RESF Partner" to description for RISC-V International.
